src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const queryList = document.querySelector('ul').children;
+
+const getEmployees = function(list) {
+  const arrayList = [...list].map(item => {
+    const itemEmployee = {
+      name: item.innerHTML.trim(),
+      position: item.getAttribute('data-position'),
+      salary: item.getAttribute('data-salary'),
+      age: item.getAttribute('data-age'),
+    };
+
+    return itemEmployee;
+  });
+
+  return arrayList;
+};
+
+const listForSort = getEmployees(queryList);
+
+const sortList = function(list) {
+  list.sort(function(a, b) {
+    let aSort = a.salary.slice(1, 10).split(',');
+
+    aSort = Number(aSort.join(''));
+
+    let bSort = b.salary.slice(1, 10).split(',');
+
+    bSort = Number(bSort.join(''));
```

Comment:
you can create a separate function for formatting values before sorting them

Reply:
DRY

Reply:
after that you can reduce your code inside sort to smth like this
```
(a, b) => toNumber(b.salary) - toNumber(a.salary)
```

Reply:
I moved the formatting into a separate function, but this syntax does not work for me